frontend/src/views/styles.module.css

```diff
@@ -0,0 +1,28 @@
+.problemPanel {
```

Comment:
nit: Move this into `Game.tsx` using [styled components](https://styled-components.com/), similar to how it's done [here](https://github.com/rocketden/main/blob/master/frontend/src/components/core/Button.tsx).

Reply:
Thoughts on also changing the styling to use **flexbox**? A guide on CSS-tricks is [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#flexbox-background); I think this would be easier to design with in the future. I'd also like to get @alankbi's thoughts on this.